config/initializers/prosody.rb

```diff
@@ -0,0 +1,3 @@
+Prosody.config({
+  file: "config/prosody.cfg.lua"
+})
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.